### DISCUSSION TITLE
Can't access so-allow and so-firewall

### DISCUSSION THREAD

**Discussion by Luxi-bit** [Security-Onion-Solutions/securityonion#11546] [2.4]
### Version

2.4.20

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

12

### Storage for /

250

### Storage for /nsm

0

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Hallo, im still learning about Security Onion, i Install into oVirt and follow all instruction from https://docs.securityonion.net/en/latest/installation.html and Youtube too. After the Installation, and i checking with command `so-status` thats nothing wrong there but when i do `so-allow` this is what i got 
<img width="482" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/61172191/6838bcfe-744b-48a9-b3ea-aba248befd8d">
`so-firewall` like this 
<img width="482" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/61172191/9a453ed7-9b42-440e-aaf8-437194764418">
I open Web Interface and i got this 
<img width="960" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/61172191/75b5d310-0238-4abd-87ae-c3f6370a29b5">
Salt Status i got 3 Error 
<img width="479" alt="Screenshot 2023-10-16 154744" src="https://github.com/Security-Onion-Solutions/securityonion/assets/61172191/d60d30d4-4f19-498c-8693-e27cb5224d88">
<img width="479" alt="Screenshot 2023-10-16 154754" src="https://github.com/Security-Onion-Solutions/securityonion/assets/61172191/6928bf78-cf5f-4f97-ad42-1aaa27ed01ef">
<img width="481" alt="Screenshot 2023-10-16 154806" src="https://github.com/Security-Onion-Solutions/securityonion/assets/61172191/9c231549-c1d8-4cf0-a48d-db036f102bee">
Can someone Help me with this ?

### Guidelines

- [x] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
12GB RAM does not meet the minimum requirement for a Standalone installation. From https://docs.securityonion.net/en/2.4/hardware.html#standalone-deployments:
> In a standalone deployment, the manager components and the sensor components all run on a single box, therefore, your hardware requirements will reflect that. You’ll need at minimum 24GB RAM, 4 CPU cores, and 200GB storage. At the bare minimum of 24GB RAM, you would most likely need swap space to avoid issues.
